### PR TITLE
Add KDE Breeze icons (boo#971671)

### PR DESCRIPTION
**Pull request by noahadvs**
~~Depends on: https://bugzilla.opensuse.org/show_bug.cgi?id=1108422~~

montage

### CODE REVIEW

File: icons/breeze/32.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+../../categories/32/applications-games.svg pattern-games.svg
+../../preferences/32/preferences-system-linux.svg pattern-kernel-devel.svg
+../../preferences/32/preferences-desktop-multimedia.svg pattern-multimedia.svg
+../../categories/32/applications-office.svg pattern-office.svg
+../../preferences/32/preferences-system-network-server.svg pattern-server.svg
+../../preferences/32/yast-software.svg pattern-software-management.svg
+../../apps/32/kwrite.svg pattern-technical-writing.svg
+../../categories/32/applications-internet.svg pattern-web-devel.svg
+yast.svg pattern-yast-devel.svg
+
```
Comment (lslezak) [inline icons/breeze/32.txt]: Empty lines at the end of the file.

File: icons/breeze/make-symlinks.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+basedir=$1
+for x in 32 48
+do
+    while read -r line; do
+        read -r -a file <<< "$line"
+        ln -sf ${file[0]} "${basedir}/apps/$x/${file[1]}"
+    done <<< `grep -v '^#' ${x}.txt`
```
Comment (lslezak) [inline icons/breeze/make-symlinks.sh]: The `.txt` suffix is usually used for text documents, here I'd use something else, e.g. `.files` or something like that.

File: package/yast2-theme.changes [hunk 3]
```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Sun Jan  6 09:36:39 UTC 2019 - Noah Davis <noahadvs@gmail.com>
+
+- Add Breeze pattern icons
```
Comment (lslezak) [inline package/yast2-theme.changes]: Any bug number to reference? Our Jenkins script would reject this change otherwise....

File: Dockerfile [hunk 4]
```diff
@@ -1,2 +1,4 @@
 FROM yastdevel/ruby
+RUN zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
+breeze5-icons
```
Comment (lslezak) [inline Dockerfile]: Please indent the line.

File: Dockerfile [hunk 5]
```diff
@@ -1,2 +1,4 @@
 FROM yastdevel/ruby
+RUN zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
```
Comment (lslezak) [inline Dockerfile]: The `--gpg-auto-import-keys` option is not needed anymore. The GPG key is already imported, this is obsolete. (In the past the key was imported by zypper, I have improved that.)

File: package/yast2-theme.changes (hunk 3 above)
Comment (noahadvs) [reply package/yast2-theme.changes]: Added it to the title

**Pull request merged**